bin/improver-wind-downscaling

```diff
@@ -73,6 +74,14 @@ def main():
                              ' Units of field: m')
     parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
                         help='The output path for the processed NetCDF')
+    parser.add_argument('--output_height_level',
+                        metavar=('OUTPUT_HEIGHT_LEVEL', 'UNITS_OF_HEIGHT'),
+                        nargs=2, default=None,
```

Comment:
I know we've used multiple arguments per option before, but I wonder if `--output_height_level_units` would be an acceptable way of avoiding it here? Default to metres in its absence would be fair enough.

Reply:
Done.